src/nbl/video/CVulkanQueue.cpp

```diff
@@ -44,7 +44,8 @@ bool CVulkanQueue::submit(uint32_t _count, const SSubmitInfo* _submits, IGPUFenc
         mem = reinterpret_cast<uint8_t*>( _NBL_ALIGNED_MALLOC(memSize, _NBL_SIMD_ALIGNMENT) );
     }
 
-    auto raii_ = core::makeRAIIExiter([mem,stackmem_]{ _NBL_ALIGNED_FREE(mem); });
+    // Todo(achal): FREE ONLY IF _NBL_ALIGNED_MALLOC was called
+    // auto raii_ = core::makeRAIIExiter([mem]{ _NBL_ALIGNED_FREE(mem); });
```

Comment:
@Crisspl didint you make a adaptive stack-but-if-too-large-heap array container in `core` ?

Reply:
Okay. Can I open an issue for it?

Reply:
sure, but i dont get the point of an issue, if its gonna get assigned straight to you.

Reply:
I want to defer it until I get the higher priority stuff is done because it is an optimization step and it would be relatively easy to replace something like `core::vector` with this container

Reply:
I'll patch up `dynamic_array` with some new capabilities like short vector optimization + on-stack creation.

For now use a `core::vector` with an initial size